modules/benchmark/R/calc.metrics.R

```diff
@@ -10,27 +10,44 @@
 ##' 
 ##' @author Betsy Cowdery
 
-calc.metrics <- function(data.path, format, model_run, metrics, start_year=NA, end_year=NA, site=NA){
+calc.metrics <- function(data.path, format, vars_used, model_run, metrics, start_year=NA, end_year=NA, site=NA){
 
   # Right now many argument are being read in as R list objects
   # This won't work remotely but at the moment I don't have a good solution
   
   # create results table
-  results <- metrics
-  results$score <- rep(NA, dim(metrics)[1])
+  results <- as.data.frame(matrix(NA, nrows = length(vars_used$pecan_name), ncol = length(metrics$name)))
+  rownames(results) <- vars_used$pecan_name
+  colnames(results) <- metrics$name
   
-  model <- read.output(model_run$id, model_run$outdir, start.year=NA, end.year=NA, var=format$mstmip_name)
-  # Names need to be mstmip variable names -- need to figure out conversion from bety names. To be done with the lookup table. 
+  # Do I need these?
+  # start.year <- lubridate::year(settings$run$start.date)
+  # end.year <- lubridate::year(settings$run$end.date)
```

Comment:
Not if you're taking them as arguments to the function. But the defaults are NA so you need to know what the default behavior of read.output is if the years are NA as well -- hopefully to just read all available output? If not, then start.year and end.year are not optional and should not have defaults